From 4d57b596ac6b7c0c10d65728472dbd6d2f460011 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 17 Feb 2015 15:30:11 +0000
Subject: [PATCH 176/462] Mantis 0034055 extension: Payment: support CCV
 OPP-C60; SCR-C

added Jars:
JarOPI13, JarCCV, JarCLAGUI, JarCUSSPayment, JarCUSSSBD, JarCUSSIllumination, JarMassaiSOAPSettings, JarMatac,

added modules
${PlatformJavaDir}/OPI_CCV20120201
${PlatformJavaDir}/SOAP-Settings


git-svn-id: svn://localhost/SelfServiceBuild/trunk@265 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 10 +++++++++-
 SelfServiceBuild/Build/java/build.xml         |  2 +-
 2 files changed, 10 insertions(+), 2 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 7ec2aa4..33df745 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -207,8 +207,13 @@
 
 <!-- MASSAI-JARs ================================================= -->
 
-<property name="JarOPI_1.3"                 value="${JarDir}/OPI_1.3.jar"/>
+<property name="JarOPI13"                   value="${JarDir}/OPI_1_3.jar"/>
+<property name="JarCCV"                     value="${JarDir}/OPI_CCV20120201.jar"/>
 <property name="JarCUSS"                    value="${JarDir}/cuss.jar"/>
+<property name="JarCLAGUI"                  value="${JarDir}/CLA_GUI.jar"/>
+<property name="JarCUSSPayment"             value="${JarDir}/CUSS.PAYMENT.XSD.jar"/>
+<property name="JarCUSSSBD"                 value="${JarDir}/CUSS.SBD.XSD.jar"/>
+<property name="JarCUSSIllumination"        value="${JarDir}/CUSS.ILLUMINATION.XSD.jar"/>
 <property name="JarMassaiConfig"            value="${JarDir}/massai-config.jar"/>
 <property name="JarMassaiControlCenter"     value="${JarDir}/massai-controlcenter.jar"/>
 <property name="JarMassaiControlCenterGate" value="${JarDir}/massai-controlcenter-gate.jar"/>
@@ -222,9 +227,12 @@
 <property name="JarMassaiPrintService"      value="${JarDir}/massai-printservice.jar"/>
 <property name="JarMassaiCLA"               value="${JarDir}/massai-cla.jar"/>
 <property name="JarMassaiCORBASettings"     value="${JarDir}/massai-corba-settings.jar"/>
+<property name="JarMassaiSOAPSettings"      value="${JarDir}/massai-soap-settings.jar"/>
 <property name="JarMassaiReleaseNotes"      value="${JarDir}/releasenotes.jar"/>
 <property name="JarMassaiStatechart"        value="${JarDir}/massai-statechart.jar"/>
 <property name="JarMatacClient"             value="${JarDir}/MATAC-CLIENT.jar"/>
+<property name="JarMatac"                   value="${JarDir}/MATAC-XBEANS.jar"/>
+
 
 <!-- Definition des IDL-Compilers: OpenORB ======================= -->
 
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index ef3e8b8..6c27b44 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -24,7 +24,7 @@
 
     <!-- Definition der Verzeichnisse mit den Java-Modulen ======= -->
     <property name="CommonDirList"   value="${CommonJavaDir}/OpenORB-1.3.1/OpenORB"/>
-    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/CLAApplet"/>
+    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/OPI_CCV20120201,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/SOAP-Settings,${PlatformJavaDir}/CLAApplet"/>
     <property name="AblsDirList"     value="${AblsJavaDir}/KioskProxy,${AblsJavaDir}/TunnelServer"/>
     <property name="AppDirList"      value=""/>
 
-- 
2.41.0.windows.1

